src/components/ContentPreview/ContentPreview.js

```diff
@@ -43,6 +43,7 @@ import '../base.scss';
 import './ContentPreview.scss';
 
 type Props = {
+    previewInstance?: any, // Allow instance of Preview library to be passed in
```

Comment:
I probably wont add it as props to keep the interface clean since this is more of a internal webapp use case for now. Just set it directly via componentRef. Maybe add a function in this file called `setPreviewInstance(preview) { this.preview = preview }`